willa/tind/fetch.py

```diff
@@ -22,7 +24,35 @@ def fetch_metadata(record: str) -> Record:
         raise RecordNotFoundError(f"Record {record} not found in TIND.")
 
     records = parse_xml_to_array(StringIO(response))
-    if len(records) > 1:
-        raise RecordNotFoundError(f"Record {record} matched more than one record in TIND.")
+    if len(records) != 1:
+        raise RecordNotFoundError(f"Record {record} did not match exactly one record in TIND.")
```

Comment:
I made this change in case an empty array is returned at a later time (because of the other API doing that).

Reply:
Added comment explaining the behaviour.